### ISSUE TITLE
cc-oci-runtime Upgrade process fails in ubuntu 

### ISSUE THREAD

**Issue by chavafg** [01org/cc-oci-runtime#810]
I tried to udpate from cc-oci-runtime 2.1.2 to 2.1.3, but the `cc-proxy.socket` cannot be restarted as `cc-proxy.service` is already started.

```
Preparing to unpack .../cc-oci-runtime_2.1.3+git.c5cfcfc-79.1_amd64.deb ...
Unpacking cc-oci-runtime (2.1.3+git.c5cfcfc-79.1) over (2.1.3+git.c5cfcfc-79.1) ...
Setting up cc-oci-runtime (2.1.3+git.c5cfcfc-79.1) ...
Job for cc-proxy.socket failed.
See "systemctl status cc-proxy.socket" and "journalctl -xe" for details.
dpkg: error processing package cc-oci-runtime (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 cc-oci-runtime
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Status of `cc-proxy.socket`:

```
Apr 10 15:12:42 ubuntu-cc-test systemd[1]: Stopping Clear Containers Proxy Socket.
Apr 10 15:12:42 ubuntu-cc-test systemd[1]: cc-proxy.socket: Socket service cc-proxy.service already active, refusing.
Apr 10 15:12:42 ubuntu-cc-test systemd[1]: Failed to listen on Clear Containers Proxy Socket.

```

The workaround is to stop `cc-proxy.service` and run the installation again.

**Comment by chavafg** [comment]
The issue also happens on Fedora: 

```
Running transaction
  Upgrading   : cc-oci-runtime-2.1.3+git.c5cfcfc-79.1.x86_64                                                                                                                                                   1/2 
Job for cc-proxy.socket failed.
See "systemctl status cc-proxy.socket" and "journalctl -xe" for details.
warning: %post(cc-oci-runtime-2.1.3+git.c5cfcfc-79.1.x86_64) scriptlet failed, exit status 1
Non-fatal POSTIN scriptlet failure in rpm package cc-oci-runtime
Non-fatal POSTIN scriptlet failure in rpm package cc-oci-runtime
  Cleanup     : cc-oci-runtime-2.1.1+git.d7de93a-75.1.x86_64                                                                                                                                                   2/2 
  Verifying   : cc-oci-runtime-2.1.3+git.c5cfcfc-79.1.x86_64                                                                                                                                                   1/2 
  Verifying   : cc-oci-runtime-2.1.1+git.d7de93a-75.1.x86_64                                                                                                                                                   2/2 

Upgraded:
  cc-oci-runtime.x86_64 2.1.3+git.c5cfcfc-79.1                                                                                                                                                                     

Complete!
```

**Comment by gorozco1** [comment]
This should be fixed in c5cfcfc-80
